releases/release-1.21/release-notes/major-themes.md

```diff
@@ -1,8 +1,8 @@
 # What's New (Major Themes)
 
-## PodIsolationPolicy over PodSecurityPolicy
+## Deprecation of PodSecurityPolicy
 
-PSP as an Admission controller resource is being deprecated. Deployed PodSecurityPolicy's will keep working until version 1.25, their target removal from the codebase. A new feature called PodIsolationPolicy is being developed in [KEP-2579](https://features.k8s.io/2579), to learn more read the featured blog [here](https://blog.k8s.io/2021/04/08/podsecuritypolicy-past-present-and-future/)
+PSP as an admission controller resource is being deprecated. Deployed PodSecurityPolicy's will keep working until version 1.25, their target removal from the codebase. A new feature, with a working title of “PSP replacement policy”, is being developed in [KEP-2579](https://features.k8s.io/2579). To learn more, read [PodSecurityPolicy Deprecation: Past, Present, and Future](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/).
```

Comment:
small nit on smart-quotes, otherwise LGTM (and thanks for updating!)

```suggestion
PSP as an admission controller resource is being deprecated. Deployed PodSecurityPolicy's will keep working until version 1.25, their target removal from the codebase. A new feature, with a working title of "PSP replacement policy", is being developed in [KEP-2579](https://features.k8s.io/2579). To learn more, read [PodSecurityPolicy Deprecation: Past, Present, and Future](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/).
```